.gitignore

```diff
@@ -1,5 +1,104 @@
-/data/
-/venv/
-/models/
-/aerosonicdb.egg-info/
-/figures/
```

Comment:
^ all of these are still included, just reshuffled into this template, some already account for in here.

It is considered good practice to use these standardizes templates as a "set and forget" measure for not including unwanted files in a Python project..

Reply:
Very nice. I'll be using this from now on. Cheers